packages/path_provider/path_provider_linux/pubspec.yaml

```diff
@@ -16,12 +16,12 @@ flutter:
         dartPluginClass: PathProviderLinux
 
 dependencies:
-  ffi: ">=1.1.2 <3.0.0"
+  ffi: ">=2.0.1"
   flutter:
     sdk: flutter
-  path: ^1.8.0
-  path_provider_platform_interface: ^2.0.0
-  xdg_directories: ">=0.2.0 <2.0.0"
+  path: ^1.8.2
+  path_provider_platform_interface: ^2.0.6
```

Comment:
Nothing in this package requires these bugfix versions; this artificially prevents clients from using earlier versions if they have a specific need to do so.